pylint/checkers/classes.py

```diff
@@ -1038,25 +1039,25 @@ def _check_attribute_defined_outside_init(self, cnode: astroid.ClassDef) -> None
             return
         defining_methods = self.config.defining_attr_methods
         current_module = cnode.root()
-        for attr, nodes in cnode.instance_attrs.items():
+        for attr, nodes_lst in cnode.instance_attrs.items():
```

Comment:
Trivial: I'd call `nodes_lst` something like `attr_nodes` instead to better describe what it is.